tests/drivers/gpio/gpio_enable_disable_interrupt/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(gpio_basic_api)
+
+FILE(GLOB app_sources src/*.c)
```

Comment:
Note - [cmake documentation](https://cmake.org/cmake/help/latest/command/file.html?highlight=CONFIGURE_DEPENDS#filesystem) discourages using GLOB.

Reply:
Updated to only add the target `src/main.c` instead of using `GLOB`.